qemu/tests/live_snapshot_basic.py

```diff
@@ -37,7 +37,6 @@ def create_image(self):
         """
         image_name = self.params.get("image_name")
         self.params['image_name_snapshot'] = image_name + "-snap"
-        image_name_snapshot = self.params['image_name_snapshot']
```

Comment:
Not really ...
The following line will use elf.params['image_name_snapshot'], not image_name_snapshot.
And I have tested it with removing the line "image_name_snapshot = self.params['image_name_snapshot']".

The case works well.